src/main/java/seedu/address/model/property/Owner.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.model.property;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Phone;
+
+/**
+ * Encapsulates all attributes for a particular Owner.
+ */
+public class Owner {
+
+    public final Name name;
+    public final Phone phone;
+
+    /**
+     * Every field must be present for an Owner to be correctly initialised.
+     */
+    public Owner(Name name, Phone phone) {
+        requireAllNonNull(name, phone);
+
```

Comment:
Maybe you can add this:
```
checkArgument(isValidPhone(phone), Phone.MESSAGE_CONSTRAINTS);
checkArgument(isValidName(name), Name.MESSAGE_CONSTRAINTS);
```

Reply:
these checks are already being done in the AddPropertyCommand::parse method, so i think its unnecessary